### ISSUE TITLE
sc2ts.initial_ts() doesn't mark 'Wuhan/Hu-1/2019' as a sample

### ISSUE THREAD

**Issue by hyanwong** [jeromekelleher/sc2ts#152]
Both flags are 0 below. Should they be?
```
>>>> print(sc2ts.initial_ts().tables.nodes)
╔══╤═════╤══════════╤══════════╤════╤════════════════════════════════════════╗
║id│flags│population│individual│time│metadata                                ║
╠══╪═════╪══════════╪══════════╪════╪════════════════════════════════════════╣
║0 │    0│        -1│        -1│   1│          {'strain': 'Vestigial_ignore'}║
║1 │    0│        -1│        -1│   0│{'date': '2019-12-26', 'strain': 'Wuh...║
╚══╧═════╧══════════╧══════════╧════╧════════════════════════════════════════╝
```

**Comment by hyanwong** [comment]
This means that the final tree sequences don't have 'Wuhan/Hu-1/2019' marked as a sample either.

**Comment by jeromekelleher** [comment]
I wasn't sure if it was really a sample, and was treating it as a special reference sequence instead. Is it?

**Comment by szhan** [comment]
It is a real sample. Also related to #102.

**Comment by jeromekelleher** [comment]
Cool, easy fix if anyone wants to push?

**Comment by hyanwong** [comment]
The fix in #156 makes various tests fail. I'm not _au fait_ with what the tests are doing, so it's not clear to me how the tests should be altered in the correct way:

```
FAILED tests/test_inference.py::TestAddMatchingResults::test_one_sample_one_mutation - AssertionError: assert 'G' == 'X'
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample_match_recombinant - AssertionError: assert 2.0 == 0
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample_one_mutation - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_two_sample_one_mutation_each - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample_k_mutations[1] - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample_k_mutations[2] - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample_k_mutations[3] - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample_k_mutations[4] - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_one_sample_k_mutations[5] - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {}
FAILED tests/test_inference.py::TestMatchPathTs::test_n_samples_metadata - AssertionError: assert {'date': '201...an/Hu-1/2019'} == {'x0': 0, 'y0': []}

```
